src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
```

Comment:
The method getRandomColor() should have an access modifier to define its visibility more clearly. By default, it's package-private, but if you intend it to be used from outside the package, it should be public.